fmskill/connection.py

```diff
@@ -295,7 +296,7 @@ def _parse_observation(self, obs) -> PointObservation:
         else:
             raise ValueError(f"Unknown observation type {type(obs)}")
 
-    def extract(self) -> PointComparer:
+    def extract(self, max_model_gap: float=None) -> PointComparer:
```

Comment:
@daniel-caichac-DHI : `.extract()` should take `max_model_gap`, no?

Reply:
I don't know, I had a script and tested with this commit and `max_gap=3600`  in `extract()` and it works,
https://github.com/DHI/fmskill/pull/148/commits/95cd56bda00a2550eab32b2e42f544e56957ec50
then I tested with the most recent version and `max_model_gap=3600`  in `extract()` and it does not connect anything :(

Reply:
Could you maybe add a test that illustrates the problem? expected vs actual...

Reply:
* When I don't use max gap I get this, 278 points
![image](https://user-images.githubusercontent.com/97288080/210729117-4065d6bb-1ab4-4b18-b31f-05caf9363c24.png)

* When use the old commit (https://github.com/DHI/fmskill/commit/95cd56bda00a2550eab32b2e42f544e56957ec50 ) I get this, 28 points
![image](https://user-images.githubusercontent.com/97288080/210729276-6f4d0bd2-56f8-4c58-9336-fa7b4941c10d.png)

* When I use the lastest commit of this branch , I get an error
![image](https://user-images.githubusercontent.com/97288080/210729426-be09b8e1-ac24-4ed2-a3f4-865628030f1d.png)

Reply:
It seems that the Connector misses this argument - only the specialized connectors, like PointConnector and TrackConnector has it.

Reply:
I just added a failing test where  `connector.extract(max_ga_mogel_gap=3600)` does not work.